### DISCUSSION TITLE
Conan Exiles Rcon

### DISCUSSION THREAD

**Discussion by link858** [gameserverapp/Platform#153] [Suggestions]
Add the ability to change what individual Rcon commands are being used by the bot
allows the user to fix the command the bot is using for in-game private messages
allows user to change command the bot is using to a custom Rcon command from MODS that enables more usability (like Pippi mod)
example 
Command
```
directmessage
                - Pippi Command - Usage: directmessage <senderName> <playerName> <message>
```
```
Rank
                - Pippi Command - Usage: rank <add||remove||list> <playerName> <rankName||rankID>
```

Some of this can be achieved by adding another task that lets you execute a console command but there is no need for that if you can tell the bot to use the correct commands for the game

**Comment by mvaessen** [comment]
duplicate of #15
